From ea6f3578b047be156c89137dc61ad874775b94b4 Mon Sep 17 00:00:00 2001
From: Andreas Sturmlechner <asturm@gentoo.org>
Date: Tue, 25 Jul 2023 15:51:40 +0200
Subject: [PATCH] Drop superfluous definition

While build fails only under certain downstream circumstances,
this line should not be here in the first place.

Follow-up to a56e12970b2e10305e2942f7aa47cc8bb9f0d53f

See also: https://invent.kde.org/education/kstars/-/issues/251

Signed-off-by: Andreas Sturmlechner <asturm@gentoo.org>
---
 kstars/skycomponents/supernovaecomponent.cpp | 2 --
 1 file changed, 2 deletions(-)

diff --git a/kstars/skycomponents/supernovaecomponent.cpp b/kstars/skycomponents/supernovaecomponent.cpp
index bb427b8e8..3b3866f7d 100644
--- a/kstars/skycomponents/supernovaecomponent.cpp
+++ b/kstars/skycomponents/supernovaecomponent.cpp
@@ -36,8 +36,6 @@
 
 #include <csv.h>
 
-gzFile gzopen OF((const char *path, const char *mode));
-
 const QString SupernovaeComponent::tnsDataFilename("tns_public_objects.csv");
 const QString SupernovaeComponent::tnsDataFilenameZip("tns-daily.csv.gz");
 const QString SupernovaeComponent::tnsDataUrl(
-- 
2.41.0

